Formula/r/rmtrash.rb

```diff
@@ -0,0 +1,22 @@
+class Rmtrash < Formula
+  desc "Move files and directories to the trash"
+  homepage "https://github.com/TBXark/rmtrash"
+  url "https://github.com/TBXark/rmtrash/archive/refs/tags/0.6.5.tar.gz"
+  sha256 "0e26ffb62b5c951728807bb3718faa2ce04bbac775ea0d5a08628f67d6319d48"
+  license "MIT"
+  head "https://github.com/TBXark/rmtrash.git", branch: "master"
+
+  depends_on xcode: ["12.0", :build]
+  depends_on :macos
+  uses_from_macos "swift", since: :big_sur
```

Comment:
this line can be removed:
- big_sur is not supported
- this formula is only for macos

```suggestion
```